pom.xml

```diff
@@ -112,12 +112,17 @@
             <dependency>
                 <groupId>org.apache.cassandra</groupId>
                 <artifactId>cassandra-all</artifactId>
-                <version>2.2.4</version>
+                <version>2.2.19</version>
```

Comment:
nitpick, consider to extract version to maven properties

Reply:
updated